DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/stats/stats.html

```diff
@@ -0,0 +1,141 @@
+                            <div class="square pull-left">
+                                <span class="glyphicon glyphicon-info-sign glyphicon-lg">{{vm.Following}}</span>
+                            </div>
+                            <h4>
+                                {{vm.i18n.message.FOLLOWING}}({{vm.Following}})
+                            </h4>
+                        </div>
+                        <div class="blockquote-box blockquote-warning clearfix">
+                            <div class="square pull-left">
+                                <span class="glyphicon glyphicon-warning-sign glyphicon-lg">
+                                    {{vm.Followers}}
+                                </span>
+                            </div>
+                            <h4>
+                                {{vm.i18n.message.FOLLOWER}}({{vm.Followers}})
+                            </h4>
+                        </div>
+                    </div>
+                    <div class="col-md-9">
+                        <hr/>
+                        <div class="panel-heading" style="background-color: #f0ad4e;">
+                            <form class="form-inline" role="form">
+                                <span class="glyphicon glyphicon-bookmark"/>
+                                    {{vm.i18n.message.STATISTICSREPOSITORYYEAR}}
+                                <div class="dropdown pull-right form-group">    
+                                    <div>
+                                        <select ng-model="repo" ng-options="repo as repo.name for repo in vm.Repositories" ng-change="vm.getCommitsFromCurrent(repo)">
+                                            <option value="">---Please select---</option>
+                                        </select>
+                                    </div>
+                                </div>
+                            </form>
+                        </div>
+                        <div class="panel panel-primary" >
+                            <div style="padding: 10px;">
+                                <canvas id="line"
+                                        class="chart chart-line" chart-data="vm.reposData"
+                                        chart-labels="vm.labels" chart-legend="true" chart-series="vm.series">
+                                </canvas>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-12">
+                        <hr/>
+                        <div class="col-md-6">
+                            <div class="panel panel-primary">
+                                <div class="panel-heading">
+                                    <h3 class="panel-title">
+                                        <span class="glyphicon glyphicon-bookmark"/>
+                                        {{vm.i18n.message.STATISTICSEACHREPOSITORYYEAR}}
```

Comment:
AAAA! compare 
`STATISTICSEACHREPOSITORYYEAR`
and
`STATISTICS_EACH_REPOSITORY_YEAR`